### PR TITLE
Domains: Update Privacy Policy upgrade to replace noticons and improve contrast

### PR DESCRIPTION
**Pull request by shaunandrews**
Replacing Noticons with Gridicons, and changing a gray background to a white background to improve text contrast.

slice

To test, you'll need a custom domain setup that *doesn't* have privacy protection enabled. Then, head to `/domains/manage/:domain/privacy-protection/:site` for your domain and site.

### CODE REVIEW

**Comment by mikeshelton1503** [comment]
Design LGTM 👍

File: client/my-sites/domains/domain-management/privacy-protection/card/content.jsx [hunk 1]
```diff
@@ -39,7 +40,7 @@ class Content extends React.PureComponent {
 				</div>
 				<div className="privacy-protection-card__features">
 					<h5>
-						<span className="noticon noticon-checkmark" />
+						<Gridicon icon="checkmark-circle" size="16" />
```
Comment (drw158) [inline client/my-sites/domains/domain-management/privacy-protection/card/content.jsx]: I thought we wrote this like `size={ 16 }`

**Comment by shaunandrews** [comment]
Ah, good call @drw158

**Comment by drw158** [comment]
Design looks good, and code looks good as far as I can tell. I think I can give you the 👍 ?

**Pull request merged**